scripts/build.sh

```diff
@@ -1,9 +1,10 @@
+#!/usr/bin/env bash
 outPath=./output
 
 rm -rf $outPath
 mkdir $outPath
 
-go build ../gogs.go
+go build ../main.go
```

Comment:
Even though the file is named `main.go` this is still incorrect, since @tboerger is correct :slightly_smiling_face: